### PR TITLE
Add support for adding subcategories description #1098

### LINKED ISSUE #1098: [Feature] Add Support for adding subcategories description
### Description

As the descriptions of Subcategories are unavailable so there should be a new category of issues for adding descriptions which will help easily attract contributors to add new ones and improve existing ones.

For doing this 2 things are needed:
1. Issue template 
2. Contributing guidelines.

### Screenshots

Screenshot_2023-06-20-17-45-30-40_40deb401b9ffe8e1df2f1cc5ba480b12

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Isha988**
## Fixes Issue
closes #1098 

## Changes proposed

1.Created a issue template
2. Updated CONTRIBUTING.md
3. updated README.md

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/category_description.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: Category Description 📝
+description: Use this to add or update category description
```
Comment (CBID2) [inline .github/ISSUE_TEMPLATE/category_description.yml:2]: [Proofreading]
```suggestion
description: Use this to add or update the category's description
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -108,6 +108,29 @@ When adding _YouTube_ channel link, please specify _the language_ of the channel
 
 ---
 
+### Adding and Updating Category Description 📝
+> ✨ You can also create [issue(s)](https://github.com/rupali-codes/LinksHub/issues/new/choose) to suggest new descriptions, and someone else will take care of them.
```
Comment (CBID2) [inline CONTRIBUTING.md:112]: [Proofreading]
```suggestion
> ✨ You can also create [issue(s)](https://github.com/rupali-codes/LinksHub/issues/new/choose) to suggest new descriptions. The maintainers will review them and then assign them.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -108,6 +108,29 @@ When adding _YouTube_ channel link, please specify _the language_ of the channel
...
+
+#### 🌟Follow these steps to add your links to the Hub
+
+- Choose the category and subcategory under which you want to add a description.
+- Locate the file: `components/TopBar/CategoryDescriptions.ts`
+- Add or update the description in front of the subcategory name.
+
+  > For example, to add description of `fonts` subcategory under `frontend` categrogy
```
Comment (CBID2) [inline CONTRIBUTING.md:120]: [Proofreading] 
```suggestion
  > For example, to add the description of the `fonts` subcategory under the `frontend` category, you would do the following: 
```

File: README.md [hunk 4]
```diff
@@ -11,6 +11,7 @@
 - [Getting Started 👩‍💻](#getting-started-)
 - [Let's jump right in🌟](#lets-jump-right-in)
 - [_Want to add your favorite links to the Hub? make sure to follow CONTRIBUTING guidelines._](#want-to-add-your-favorite-links-into-the-hub-make-sure-to-follow-contributing-guidelines)
+- [Add or update description](#want-to-add-or-update-the-descriptions-of-subcategroies)
```
Comment (CBID2) [inline README.md:14]: [Typo]
```suggestion
- [Add or update description](#want-to-add-or-update-the-descriptions-of-subcategories)
```

File: README.md [hunk 5]
```diff
@@ -124,6 +125,9 @@ You can see the live demo at: https://linkshub.vercel.app
 
 #### _Want to add your favourite links to the Hub? make sure to follow [CONTRIBUTING guidelines](https://github.com/rupali-codes/linkshub/blob/main/CONTRIBUTING.md)._
 
+#### Want to add or update the descriptions of subcategroies?
```
Comment (CBID2) [inline README.md:128]: [Typo]
```suggestion
#### Want to add or update the descriptions of subcategories?
```

File: README.md [hunk 6]
```diff
@@ -124,6 +125,9 @@ You can see the live demo at: https://linkshub.vercel.app
...
+make sure to follow [CONTRIBUTING guidelines](./CONTRIBUTING.md#Adding-and-Updating-Category-Description-).
```
Comment (CBID2) [inline README.md:129]: [Proofreading]
```suggestion
Make sure to follow [CONTRIBUTING guidelines](./CONTRIBUTING.md#Adding-and-Updating-Category-Description-).
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Isha988! :) I pointed out some typos and proofreading errors in my inline comments. Other than that, you're off to a great start! :)

**Comment by Isha988** [comment]
@CBID2 I have made all the required changes

**Review by CBID2 — APPROVED** [review]
LGTM

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**